vip-helpers/vip-migrations.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+add_action( 'vip_after_data_migration', 'vip_after_data_migration' );
+
+function vip_after_data_migration() {
+	if ( is_multisite() ) {
+		$sites = get_sites();
+
+		foreach( $sites as $site ) {
+			switch_to_blog( $site->blog_id );
+
+			vip_run_after_data_migration_cleanup();
+
+			restore_current_blog();
+		}
+
+		return true;
+	} else {
+		return vip_run_after_data_migration_cleanup();
+	}
+}
+
+function vip_run_after_data_migration_cleanup() {
+	/**
+	 * Fires on migration cleanup
+	 *
+	 * Migration cleanup runs on VIP Go during the initial site setup
+	 * and after database imports. This hook can be used to add additional
+	 * cleanup for a given site.
+	 */
+	do_action( 'vip_go_migration_cleanup' );
+
+	vip_delete_db_transients();
```

Comment:
Should we just hook these functions into `vip_go_migration_cleanup` instead? It would simplify the wp-cli as well as get rid of the duplication. For the dry run, we could just list out the callback names that would fire (or just remove the dry-run).

Reply:
IMO it's ok to keep them separate - the CLI command has a slightly different use case as it is meant to also cleanup jp/vp options (which the cron event doesn't as that's already happened). I can imagine it could diverge further in the future.

Reply:
Moved it to the same func that's called by cron in https://github.com/Automattic/vip-go-mu-plugins/pull/841/commits/a04c34c80a944813ec0cc49007556278e6cccbf7